app/vmalert/rule/alerting_test.go

```diff
@@ -675,7 +675,7 @@ func TestGroup_Restore(t *testing.T) {
 	// two rules, two active alerts, one with state restored
 	ts = time.Now().Truncate(time.Hour)
 	fqr.Set(`last_over_time(ALERTS_FOR_STATE{alertgroup="TestRestore",alertname="bar"}[3600s])`,
-		stateMetric("foo", ts))
+		stateMetric("bar", ts))
```

Comment:
this is weird... why did it work before? is this test broken?

Reply:
Previous we didn't check result's labelset hash cause we didn't need to match the result with `ar.alerts`, so the name was wrong but do nothing.